FxConsole/Program.cs

```diff
@@ -21,6 +21,8 @@ static void Main(string[] args)
 			Console.WriteLine("enter any key to begin...");
 			Console.ReadLine();
 
+			var y = +1;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'y'.](https://app.codacy.com/manual/lspangler584/FxConsole/pullRequest?prid=4751234)